### PR TITLE
Add a new piped image dedicated to OKD

### PR DESCRIPTION
**Pull request by nakabonne**
**What this PR does / why we need it**:
This PR makes it available to use a brand new independent piped packing with `nss_wrapper`. Aside from making a new directory underneath `cmd/`, if someone knows anything else way, tell me what to do.

**Which issue(s) this PR fixes**:

Ref https://github.com/pipe-cd/pipe/issues/1905

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -135,6 +135,15 @@ container_pull(
     tag = "0.2.2",
 )
 
+container_pull(
+    name = "piped-base-with-nss-wrapper",
+    # FIXME: Populate digest after pushing piped-base-with-nss-wrapper
+    digest = "",
```
Comment (nakabonne) [inline WORKSPACE:141]: looking to populate once https://github.com/pipe-cd/pipe/pull/1952 gets merged.

**Comment by nakabonne** [comment]
Thanks to https://github.com/pipe-cd/pipe/pull/1962, multiple image build is now available. Looking to use it.

**Comment by nakabonne** [comment]
Got stuff done 👍

**Comment by nghialv** [comment]
We have to update this file too.
https://github.com/pipe-cd/pipe/blob/master/cmd/image.bzl#L3

```
"piped": "piped_okd",
```

**Comment by nakabonne** [comment]
Oops

File: dockers/piped-base-okd/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# piped-base-okd
+This directory contains the Piped docker image packing with [nss_wrapper](https://cwrap.org/nss_wrapper.html) to add an random UID to "passwd" at runtime without having to directly modify `/etc/passwd`.
```
Comment (nghialv) [inline dockers/piped-base-okd/README.md:2]: nit: `a random`

Comment (nghialv) [reply dockers/piped-base-okd/README.md:2]: nit: `add the logged-in UID` is better

**Comment by nakabonne** [comment]
>(07:28:21) ERROR: package contains errors: cmd

**Comment by nakabonne** [comment]
>Error: dictionary expression has duplicate key: "piped"

I'm in the middle of looking up but seems like unable to use the same key.

**Comment by nghialv** [comment]
Haha. The same key in that dictionary.

**Comment by nakabonne** [comment]
solved out with a little bit dirty way.

**Comment by khanhtc1202** [comment]
🚀 
/approve

**Pull request merged**